examples/tc-egress/README.md

```diff
@@ -6,23 +6,10 @@
 1. Install a rust nightly toolchain: `rustup install nightly`
 1. Install bpf-linker: `cargo install bpf-linker`
 
-## Build eBPF
+## Build & Run
 
-```console
-cargo xtask build-ebpf
-```
-
-To perform a release build you can use the `--release` flag.
-You may also change the target architecture with the `--target` flag
-
-## Build Userspace
-
-```console
-cargo build
-```
-
-## Run
+Use `cargo build`, `cargo check`, etc. as normal. Run your program with:
 
-```console
-cargo xtask run
+```shell
+cargo run --config 'target."cfg(all())".runner="sudo -E"'
```

Comment:
Let's keep the `RUST_LOG=info` thing in every example? We should also add it in the template. The reason why it was added in the first place was amount of confused people asking on Discord "but why I can't see the logs?" :sweat_smile:

Reply:
Will do. I want to get clarify on `aya-build` before merging this pile of duplicated code :(

Reply:
Done.